pyproject.toml

```diff
@@ -0,0 +1,46 @@
+[tool.codespell]
+count = ""
+ignore-regex = '\b[A-Z]{3}\b'
+ignore-words-list = "ans,asend,deques,dout,extint,iput,ure"
+quiet-level = 3
+skip = [
+  ".*_cache",
+  "./drivers/cc3100",
+  "./lib",
+  "./ports/cc3200/hal",
+  ".git",
+  "**/build*",
+]
+
+[tool.ruff]
+exclude= [
+  "ports/cc3200/tools/uniflash.py",
+  "tests",
+]
+extend-select = ["C9", "PLC"]
+ignore = [
+  "E401",
+  "E402",
+  "E501",
+  "E711",
+  "E712",
+  "E713",
+  "E721",
+  "E722",
+  "E731",
+  "E741",
+  "E999",
+  "F401",
+  "F403",
+  "F524",
+  "F541",
+  "F632",
+  "F811",
+  "F821",
+  "F841",
```

Comment:
Will these ignores gradually be removed once the problems are fixed?

Reply:
✅

Reply:
Small is good.  As long as they are forward steps!